optuna/integration/botorch.py

```diff
@@ -652,7 +652,7 @@ def sample_independent(
     def reseed_rng(self) -> None:
         self._independent_sampler.reseed_rng()
         if self._seed is not None:
-            self._seed = numpy.random.RandomState().randint(2**60)
+            self._seed = numpy.random.RandomState().randint(numpy.iinfo(numpy.int32).max)
```

Comment:
The `2**60` is out of range of 32bit integers. Should we use `numpy.int64`?

Reply:
There are no particular reasons to use `2**60` as the argument, so it is enough to use `numpy.int32`.